internal/artifacts.go

```diff
@@ -276,6 +276,21 @@ func (b *ArtifactsBuilder) Build() (*Artifacts, error) {
 			}
 		}
 
+		// Update the allocs to include the same prefunded accounts as the L1 genesis.
+		allocs := make(map[string]interface{})
+		input["alloc"] = allocs
+		for _, privStr := range prefundedAccounts {
```

Comment:
We could hardcode these into the `genesis.json` if you prefer. Felt a bit neater to duplicate it here so the addresses we fund don't get out of sync.